src/libexec/zmdiaglog

```diff
@@ -607,6 +607,7 @@ sub run() {
         my @network_conf_files = (
             "/opt/zimbra/conf/zextras/config",
             "/opt/zimbra/conf/zextras/oplog",
+            "/opt/zimbra/conf/zextras/bulk_delete/*",
```

Comment:
I'm not sure what we would typically find under here, but if we're trying to get files under bulk_delete, then we need to use `glob()` similar to two lines below.

Also, unrelated, but curious... why is a log (`oplog`) living under /opt/zimbra/conf (and `/opt/zimbra/conf/zextras/data/` too)?  Logs/data files are typically under /var/log or /opt/zimbra/log or /opt/zimbra/data, but not under config.  Sigh.

Reply:
@DinoPuller  can you please address review comments?

Reply:
@plobbes The files in the /opt/zimrba/conf/zextras/* are controlled by zxtras. They are separate because they are not part of the core zimbra product. I don't like it either, but that's where they are.

bulk_delete contains a json file with a list of message ids that zextras will delete based on a sweeper process. The sweeper will read the file then delete those blobs. this is how they prevent stale blobs from being in the store, and also prevent large delete requests from impacting the server.

We could ask them to move the files, no problem with that. but the files added by @DinoPuller could be useful, as we had a case today where I needed them.

If you can, anything recursively in the /opt/zimbra/conf/zextras dir would be incredibly useful for zmdialog.